.github/workflows/swg-cli.yaml

```diff
@@ -0,0 +1,20 @@
+name: Test GH Action
+on:
+  push:
+    branches:
+      - master
+      
+env:
+  SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: A job to verify action
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Call CLI
+      id: swg-cli
+      uses: michaelmelody91/swaggerhub-cli@pr_demo
+      with:
+        args: api:get michaelmelody/SimpleApi
```

Comment:
Not to be included. Will remove after review.

Reply:
Cheers @michaelmelody91 I can merge once you update the PR.

Reply:
Done, thanks @barryadk!